### ISSUE TITLE
Breadcrumbs inconsistently in the main element

### ISSUE THREAD

**Issue by aduggin** [alphagov/static#902]
The breadcrumbs are inside the main element on the newer layouts but the older layouts have them outside the main element, this could be an accessibility issue since the skip link skips to the main to avoid a bunch of navigation.

cc @nickcolley (written by @nickcolley 🐒 )

- https://www.gov.uk/government/collections/statutory-guidance-schools#administration-and-finance
- https://www.gov.uk/become-lorry-bus-driver/applying-for-a-bus-or-lorry-provisional-entitlement

Publishing Frontend Support: https://trello.com/c/KdTJIBdA/37-a11y-breadcrumbs-inconsistently-in-the-main-element

**Comment by fofr** [comment]
Which one is correct?

**Comment by nickcolley** [comment]
There was suggestion that the breadcrumbs should be skipped but I'm not certain.

**Comment by nickcolley** [comment]
@alphagov/accessibility can you confirm the breadcrumbs should be skipped? 😄

**Comment by selfthinker** [comment]
Yes, the breadcrumbs should be skipped. They should also not be part of `main` for other reasons, so just moving the `id` will probably not be enough.
The main content usually starts where the `h1` is, it's the main content people would be interested to get to easily. Breadcrumbs are more some kind of navigation, not content.

**Comment by andysellick** [comment]
Page layouts have been updated so now the breadcrumb should always be outside of the `main` element as suggested.
